### ISSUE TITLE
Feature Request: Parameterize systems/downloads in yaml file 

### ISSUE THREAD

**Issue by privateemu** [hadobedo/Myrient-Downloader-GUI#1]
I was looking for something like this to make downloading a bunch of items far easier, thank you for making it! The main issue I'm having now is that some of the systems I want to download are not part of the current code. I was looking through the python code and while adding one or two more is not terrible, it does require updating the code in a dozen or so places. I was thinking that perhaps this might be best optimized by storing everything in a yaml file that one could for now just have boolean true/false for which ones they want to download from, or a selection screen when starting up for what to choose. And then parameterize the code to dynamically add the code/buttons depending on what's selected.

**Comment by hadobedo** [comment]
You're absolutely right, the 'sources'/system downloads should be stored in a separate yaml file, doing this would both directly and undirectly lead to cleaner code too! I'm gonna start chipping away at this script again pretty soon with a focus on making the software much more modular :)

**Comment by MrArdillo** [comment]
Some news for support for another systems like microsoft and Nintendo consoles?

**Comment by hadobedo** [comment]
v0.4.0 is now out with parameterized yaml, no more hardcoded urls!! to add more systems to the GUI in v0.4.0, take a peek at config/myrient_urls.yaml` and you'll see how the URL's are defined, I'll update the readme ASAP but should hopefully be pretty self-explanatory for now 🙏

**Comment by privateemu** [comment]
Woo! Thank you! I'll check this out!
